src/PrizePool.sol

```diff
@@ -240,11 +251,13 @@ contract PrizePool is TieredLiquidityDistributor {
         }
         prizeToken = params.prizeToken;
         twabController = params.twabController;
-        drawManager = params.drawManager;
         smoothing = params.smoothing;
         claimExpansionThreshold = params.claimExpansionThreshold;
         drawPeriodSeconds = params.drawPeriodSeconds;
         _lastCompletedDrawStartedAt = params.firstDrawStartsAt;
+
+        drawManager = params.drawManager;
+        emit DrawManagerSet(params.drawManager);
```

Comment:
So we keep the possibility of setting the manager in the constructor?
If we want to keep the possibility of setting it through `setDrawManager`, we will need to pass `address(0)`.
If `address(0)` is passed, we should not emit the event `emit DrawManagerSet(params.drawManager);`